src/pudl/etl/__init__.py

```diff
@@ -73,6 +73,7 @@
     *load_assets_from_modules(
         [pudl.analysis.plant_parts_eia], group_name="plant_parts_eia"
     ),
+    *load_assets_from_modules([pudl.analysis.ferc1_eia], group_name="ferc1_eia_stuff"),
```

Comment:
What if we combined all the assets that are currently part of the `plant_parts_eia` group, and the new one from this group into a single `ferc1_eia_record_linkage` or `ferc1_eia_entity_matching` group, since that's what they're all aimed at conceptually, and that's more intelligible to an outsider than `plant_parts_list` and we anticipate integrating additional tables within that theme into the database from the work that's already been done in the RMI repo.

There's also clear dependencies between these big tables, and in development we're probably going to need to regenerate them together, or one at a time as we futz with things, and being able to display them all and work with them in a single asset group will be nice.

```py
*load_assets_from_modules(
    [
        pudl.analysis.plant_parts_eia,
        pudl.analysis.ferc1_eia,
    ],
    group_name="ferc1_eia_record_linkage",
)
```